cli/cmd/config/read/command_test.go

```diff
@@ -0,0 +1,138 @@
+package read
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/consul-k8s/cli/common"
+	cmnFlag "github.com/hashicorp/consul-k8s/cli/common/flag"
+	"github.com/hashicorp/consul-k8s/cli/common/terminal"
+	"github.com/hashicorp/consul-k8s/cli/helm"
+	"github.com/hashicorp/go-hclog"
+	"github.com/posener/complete"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
+	helmRelease "helm.sh/helm/v3/pkg/release"
+	helmTime "helm.sh/helm/v3/pkg/time"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func TestConfigRead(t *testing.T) {
+	nowTime := helmTime.Now()
+	cases := map[string]struct {
+		messages           []string
+		helmActionsRunner  *helm.MockActionRunner
+		expectedReturnCode int
+	}{
+		"empty config": {
+			messages: []string{"\n"},
+
+			helmActionsRunner: &helm.MockActionRunner{
+				GetStatusFunc: func(status *action.Status, name string) (*helmRelease.Release, error) {
+					return &helmRelease.Release{
+						Name: "consul", Namespace: "consul",
+						Info:   &helmRelease.Info{LastDeployed: nowTime, Status: "READY"},
+						Chart:  &chart.Chart{Metadata: &chart.Metadata{Version: "1.0.0"}},
+						Config: make(map[string]interface{})}, nil
+				},
+			},
+			expectedReturnCode: 0,
+		},
+		"some config": {
+			messages: []string{"\n"},
+
+			helmActionsRunner: &helm.MockActionRunner{
+				GetStatusFunc: func(status *action.Status, name string) (*helmRelease.Release, error) {
+					return &helmRelease.Release{
+						Name: "consul", Namespace: "consul",
+						Info: &helmRelease.Info{LastDeployed: nowTime, Status: "READY"},
+						Chart: &chart.Chart{
+							Metadata: &chart.Metadata{
+								Version: "1.0.0",
+							},
+						},
+						Config: map[string]interface{}{"global": "true"},
+					}, nil
+				},
+			},
+			expectedReturnCode: 0,
+		},
+	}
+	for name, tc := range cases {
+		t.Run(name, func(t *testing.T) {
+			buf := new(bytes.Buffer)
+			c := getInitializedCommand(t, buf)
+			c.kubernetes = fake.NewSimpleClientset()
+			c.helmActionsRunner = tc.helmActionsRunner
+			returnCode := c.Run([]string{})
+			require.Equal(t, tc.expectedReturnCode, returnCode)
+			output := buf.String()
+			for _, msg := range tc.messages {
+				require.Contains(t, output, msg)
+			}
+		})
+	}
+}
+
+func TestTaskCreateCommand_AutocompleteFlags(t *testing.T) {
```

Comment:
Curious, are changes in this PR to prediction behavior?  It feels like prediction tests should not he to go into ever command if predictions are global and the command does not do anything new.  

(not looking for a code change, but just curious if these are here because other commands had them)

Reply:
Yeah most of the other commands have this boilerplate in it. I think it could be fine to keep or remove in this PR, but probably we should have a separate task to pull out the autocomplete code elsewhere